daskernetes/objects.py

```diff
@@ -86,20 +86,28 @@ def merge_dictionaries(a, b, path=None, update=True):
     return a
 
 def make_pod_spec(
-        image,
+        image='daskdev/dask:latest',
         labels={},
         threads_per_worker=1,
         env={},
         extra_container_config={},
         extra_pod_config={},
-        memory_limit=None,
-        memory_request=None,
-        cpu_limit=None,
-        cpu_request=None,
+        memory_limit='4GB',
+        memory_request='4GB',
+        cpu_limit=1,
+        cpu_request=1,
```

Comment:
Is it normal to provide resource defaults, or is it more typical to let these default to `None`?  I'm inclined to add limits, at least for memory.  Also, what are valid values here?  `G`, `GB`, `GiB` ?

Reply:
yeah, it's normal to default those to None.